ui/src/app/pages/data/cohort/attributes-page.spec.tsx

```diff
@@ -193,29 +188,33 @@ describe('AttributesPageV2', () => {
   });
 
   it('should render a single input for EQUAL operator and disable calculate button when empty', async () => {
-    const wrapper = component();
-    const numericalDropdown = getNumericalDropdown(wrapper, '0');
-    numericalDropdown.props.onChange({
-      originalEvent: undefined,
-      value: Operator.EQUAL,
-      target: { id: '', name: '', value: Operator.EQUAL },
-      stopPropagation: () => {},
-      preventDefault: () => {},
-    });
-    await waitOneTickAndUpdate(wrapper);
-    expect(wrapper.find('[id="numerical-input-0-0"]').length).toBe(2);
-    expect(
-      wrapper.find('[data-test-id="attributes-calculate-btn"]').first().props()
-        .disabled
-    ).toBeTruthy();
-    const numericalInput = getNumericalInput(wrapper, '0');
-    numericalInput.simulate('change', { target: { value: 100 } });
-    await waitOneTickAndUpdate(wrapper);
-    expect(
-      wrapper.find('[data-test-id="attributes-calculate-btn"]').first().props()
-        .disabled
-    ).toBeFalsy();
-    wrapper.find('[data-test-id="attributes-calculate-btn"]').simulate('click');
-    expect(mockCountParticipants).toHaveBeenCalledTimes(1);
+    const user = userEvent.setup();
```

Comment:
same Q for this test

Reply:
Do you want me to dig into this some more, or does the thread above suffice for this test?

Reply:
In the old code, Line 197 seems equivalent to line 92. I'm not sure why line 92 did not call `getNumericalDropdown(wrapper, '0');`

Line 198-204 (old code) triggers a change  event on the dropdown. Lines 196-201 in the new code triggers the same change but does it by clicking on the dropdown and then clicking on the "EQUALS" option.